packages/nextjs-contentful-renderer/src/MdxRenderer/MdxRenderer.tsx

```diff
@@ -15,3 +15,6 @@ export const MdxRenderer: FC<MdxRendererProps> = ({ content = {}, componentMap =
 
   return !isClient ? null : <MdxComponent components={{ ...components, ...componentMap }} />;
 };
+
+export * from './MarkdownContext';
+export * from './MarkdownContext/types';
```

Comment:
@tkanzakic Is there a better way to do this? Basically, I need to make sure this gets included in the `@cmpsr/nextjs-contentful-renderer/client` exports.

Reply:
do we have to export the types from here too?, they are already exported in the index.ts file

Reply:
Whoops, something I missed while moving stuff around. Should be good now.